Directory.Build.props

```diff
@@ -22,8 +23,8 @@
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <EnableNETAnalyzers>true</EnableNETAnalyzers>
+    <AnalysisMode>Recommended</AnalysisMode>
     <Nullable>enable</Nullable>
-    <IsPackable>true</IsPackable>
```

Comment:
This must be reverted.  Conditions that rely on `IsPackable` do not work properly without it.

Reply:
I thought about it. It looks strange, especially since before everything seemed to work properly with maybe the exception of Readme not showing on nuget.org, which we simply may not notice. The same doubts about all other projects. I changed some conditions around `IsPackable`. Only one condition left that should work for `ItemGroup`:
```
<ItemGroup Condition="'$(IsPackable)' == 'true'">
    <None Include="..\..\assets\logo.64x64.png" Pack="true" PackagePath="\" />
    <None Include="..\..\README.md" Pack="true" PackagePath="\" />
    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
  </ItemGroup>
```

Reply:
True, but this is confusing.  Nobody is going to understand in the future why `'$(IsPackable)' == 'true'` works in one spot and not another.  And if anyone does need a conditional on a `PropertyGroup` item, they will not be able to do and would need to revert these changes.

In addition, we have a consistent build process for the GraphQL.NET project, this project and the relay project.  It works well.  We should not make a change here that differs from the other projects.

Please revert this change.

Reply:
I reverted it a bit but I can not agree 100% that I like current status quo.